aliveim.go

```diff
@@ -56,7 +62,11 @@ func handleAlivePost(rw http.ResponseWriter, request *http.Request) {
 	} else {
 		timer := time.NewTimer(time.Millisecond * time.Duration(aliverequest.Timeout))
 		deviceTimer := DeviceTimer{aliverequest.DeviceID, timer}
+
+		mutex.Lock()
 		timersMap[aliverequest.DeviceID] = deviceTimer
+		mutex.Unlock()
+
 		go deviceTimer.startTimer()
```

Comment:
Mutex should be released when access to the data you're locking could be safely allowed to other callers. It wouldn't make sense to use a mutex at all otherwise